packages/client-config/src/client-config-contributor/auth_client_config_contributor.test.ts

```diff
@@ -43,9 +43,54 @@ void describe('AuthClientConfigContributor', () => {
         },
       }),
       {
+        aws_user_pools_id: 'testUserPoolId',
+        aws_user_pools_web_client_id: 'testWebClientId',
         aws_cognito_region: 'testRegion',
+        aws_cognito_identity_pool_id: 'testIdentityPoolId',
+      }
+    );
+  });
+
+  void it('returns translated config when output has auth with zero-config attributes', () => {
+    const contributor = new AuthClientConfigContributor();
+    assert.deepStrictEqual(
+      contributor.contribute({
+        [authOutputKey]: {
+          version: '1',
+          payload: {
+            identityPoolId: 'testIdentityPoolId',
+            userPoolId: 'testUserPoolId',
+            webClientId: 'testWebClientId',
+            authRegion: 'testRegion',
+            passwordPolicyMinLength: '15',
+            passwordPolicyRequirements:
+              '["REQUIRES_NUMBERS","REQUIRES_LOWERCASE","REQUIRES_UPPERCASE"]',
+            mfaTypes: '["SMS","TOTP"]',
+            mfaConfiguration: 'OPTIONAL',
+            verificationMechanisms: '["EMAIL","PHONE"]',
+            usernameAttributes: '["EMAIL"]',
+            signupAttributes: '["EMAIL"]',
+          },
+        },
+      }),
+      {
         aws_user_pools_id: 'testUserPoolId',
         aws_user_pools_web_client_id: 'testWebClientId',
+        aws_cognito_region: 'testRegion',
+        aws_cognito_identity_pool_id: 'testIdentityPoolId',
+        aws_cognito_mfa_configuration: 'OPTIONAL',
+        aws_cognito_mfa_types: ['SMS', 'TOTP'],
+        aws_cognito_password_protection_settings: {
+          passwordPolicyCharacters: [
+            'REQUIRES_NUMBERS',
+            'REQUIRES_LOWERCASE',
+            'REQUIRES_UPPERCASE',
```

Comment:
I'm assuming we've confirmed with the library team that these are the values they are expecting?

Reply:
These are the values from gen 1 CLI, they should be the same